schema/filespec-schema.json

```
@@ -42,6 +42,19 @@
           }
         }
       },
+      "pathMapping": {
+        "description": "If specified, determines release-bundle source and target paths for artifacts that had been fetched using aql",
+        "properties": {
+          "input": {
+            "type": "string",
+            "description": "The input path for mapping."
```

Configuring `pathMapping` is challenging and the documentation does not explain how to do it. Let's add an example:
```suggestion
            "description": "The input path for mapping.",
            "examples": ["(.*)/old_folder/(.*)"]
```